draft-tus-resumable-uploads-protocol.md

```diff
@@ -196,7 +196,7 @@ If the offset in the `Upload-Offset` header field does not match the value 0, th
 
 If the request completes successfully and the entire upload is complete, the server MUST acknowledge it by responding with a successful status code between 200 and 299 (inclusive). Server is RECOMMENDED to use `201 (Created)` response if not otherwise specified. The response MUST NOT include the `Upload-Incomplete` header with the value of true.
 
-If the request completes successfully but the entire upload is not yet complete indicated by the `Upload-Incomplete` header, the server MUST acknowledge it by responding with the `201 (Created)` status code and the `Upload-Incomplete` header set to true.
+If the request completes successfully but the entire upload is not yet complete indicated by the `Upload-Incomplete` header, the server MUST acknowledge it by responding with the `201 (Created)` status code, the `Upload-Incomplete` header set to true, and the `Upload-Offset` header set to the new upload resumption offset.
```

Comment:
What do you think about lowering this to a SHOULD or MAY? Since the lack of this header does not affect the upload.

Reply:
I prefer it if the spec leaves less room for such optional behavior. Especially in this case, since the server should have no problems complying with this requirement.

Reply:
That makes sense. If it helps with debugging, do you think this requirement should be moved to all Upload Transfer Procedure responses, rather than just the incomplete ones?